package.json

```diff
@@ -2,20 +2,42 @@
   "name": "user_reg",
   "version": "0.1.0",
   "private": true,
+  "homepage": "http://srkirve77.github.io/user_reg/src",
   "dependencies": {
+    "@babel/preset-env": "^7.14.1",
+    "@babel/preset-react": "^7.13.13",
+    "@babel/runtime": "^7.12.1",
+    "@material-ui/core": "^4.11.4",
+    "@material-ui/icons": "^4.11.2",
     "@testing-library/jest-dom": "^5.12.0",
     "@testing-library/react": "^11.2.6",
     "@testing-library/user-event": "^12.8.3",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-scripts": "4.0.3",
+    "chai": "^4.3.4",
```

Comment:
are we using `chai`?

Reply:
Yes sir.I was using chai but then in code i just switched to default expect.i will remove this